cloudpickle/cloudpickle.py

```diff
@@ -278,6 +278,20 @@ def __init__(self, file, protocol=None):
         # map ids to dictionary. used to ensure that functions can share global env
         self.globals_ref = {}
 
+        # When unpickling a function created in a dynamic module or the __main__
+        # module, its set of global variables may be different from the one from
+        # the module the function belongs to, for example if the dynamic module
+        # has had its variables mutated between pickling and unpickling time
+        # of the function.
+
+        # - If set to True, the attribue override_existing_globals will
```

Comment:
```python
        # - If override_existing_globals=True, the global variables of the module
        # containing this function will be overridden in the process which load
        # the pickled object with the values of these variables at the pickling
        # time.
        # - If override_existing_globals=False, the global variables of the module
        # containing this function will not be overridden in the process which load
        # the pickled object. If the global variable is not declared, it will be
        # initialized with the value at pickling time.
```